sql/normandy_recipe_versions_v1.sql

```diff
@@ -0,0 +1,14 @@
+SELECT
+  f.key AS recipe_id,
+  f.value AS recipe_revision_id,
+  COUNT(f.value) AS count
+FROM
+  `moz-fx-data-derived-datasets.telemetry.main_summary_v4` m,
+  m.scalar_parent_normandy_recipe_freshness.key_value f
```

Comment:
I'm pretty sure this is missing an `UNNEST`: 
```suggestion
  UNNEST(m.scalar_parent_normandy_recipe_freshness.key_value) f
```

Reply:
Could you tell me more about why? I was working from example queries in https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax#syntax, which shows a `SELECT` query which queries an array field directly.

Reply:
you're totally right, i just hadn't seen that it was an option >_<